Fix hyperlinks and add a limitation. (#184)
GeyserMC/GeyserWiki: _docs/geyser/current-limitations.md

- Various command arguments for any command that doesn't use the Minecraft Brigadier library
- Anything that relies on tab complete or typing in the chat UI (related to the above) - Bedrock sends no packet that indicates they are in this menu
- Unable to see banner layers past 6

> - Movement issues around bamboo due to offset differences between Java and Bedrock. The following plugin works around this, but please read the README before using: [Hurricane](/other/hurricaneh)

Reviewer: ```suggestion
- Movement issues around bamboo due to offset differences between Java and Bedrock. [Hurricane](/other/hurricane) can work around it.
```

The warning itself is on the page, plus there was a typo

- Various command arguments for any command that doesn't use the Minecraft Brigadier library
- Anything that relies on tab complete or typing in the chat UI (related to the above) - Bedrock sends no packet that indicates they are in this menu
- Unable to see banner layers past 6

> - Movement issues around bamboo due to offset differences between Java and Bedrock. The following plugin works around this, but please read the README before using: [Hurricane](https://github.com/GeyserMC/Hurricane)

Author: You're right, I'll make changes。

> - Can't move mounted entities with carrot/warped fungus on a stick(1.17.x & above). Hurricane also works around this.

Reviewer: I'd remove the version here, and instead add e.g. camels to this list - Camels specifically are not fixed by Hurricane though.
